recipe/build.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
```

Comment:
So `libedit` is a different library that provides a `readline`-like API, but is more limited in supported features. Think of it as a BSD `readline`. This is being picked up from `/usr/lib/libedit.3.dylib`. Mac ships with it as they prefer not to have `readline` due to the GPL.

We probably need some heavy duty path manipulation to make sure it doesn't search there. My last attempt managed to link against `libedit`, `readline`, and `ncurses`, but I'm not sure whether it was actually using symbols from the latter two or just adding them to the rpath.